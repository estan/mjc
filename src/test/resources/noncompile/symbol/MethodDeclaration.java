class MethodDeclarationTest {
    public static void main(String[] args) {
    }
}
class A {
    public int f(int a, boolean b) {
        return 42;
    }
    public int[] f(boolean a, int b) { // DUPLICATE_METHOD
        int[] ia;
        return ia;
    }
    public B g() { // UNDECLARED_CLASS
        return 42;
    }
}
