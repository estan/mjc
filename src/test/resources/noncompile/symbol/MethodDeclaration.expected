DUPLICATE_METHOD
UNDECLARED_CLASS
